beater/pub.go

```diff
@@ -16,21 +21,26 @@ import (
 // number requests(events) active in the system can exceed the queue size. Only
 // the number of concurrent HTTP requests trying to publish at the same time is limited.
 type publisher struct {
-	events  chan []beat.Event
+	events  chan event
 	client  beat.Client
 	m       sync.RWMutex
 	stopped bool
 }
 
+type event struct {
```

Comment:
since we have already error events, transactions events, and beat events, would be nice to unclog a bit our names and call this differently...